src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,46 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+/**
+ * Command: Deletes the Task given
+ */
+public class DeleteCommand extends Command {
+    private int taskNo;
```

Comment:
As taskNo might be informal english and could be confused with the word "No", perhaps the variable name taskNo could be changed to taskNumber?